### DISCUSSION TITLE
臉書誤將黑人辨識為"靈長類動物"

### DISCUSSION THREAD

**Discussion by jerry850313** [1101-GS4719-C/Python-Community-GS4719-C#6] [General]
以下是我對於神經網路的了解 :
所謂層(layer)就是電腦將資訊整理成他所知道的內容的一個過程。
層數則是經過多少次整理，通常2層以上可被稱作深度學習(deep learning)。
我覺得這個問題的主因可能是在神經網路上的層(layer)相關算法或權重，
導致系統認為影片中的黑人符合它所認知的大部分靈長類特徵吧?
[新聞來源](https://reurl.cc/vg19nl)

**Comment by enoch-lightning** [comment]
我覺得您所謂的「神經網路」，會將黑人分成「靈長類動物」是非常合理的。因為人類本來就被歸類成靈長類動物。照您這樣說，facebook是可以特別從「靈長類動物」中將人類分別出來嗎？除了黑人以外，其他人種，包括亞洲人與白種人都不會被判斷成「靈長類動物」嗎？是不是因為其他的靈長類動物，膚色都較深，導致黑人被判斷成靈長類動物？

**Reply by jerry850313** [reply]
我覺得靈長類可能只是第一層
第二層之後的判斷有問題
才導致這個結果
再來就是權重分配上也是原因之一

**Reply by enoch-lightning** [reply]
初步來看，我覺得你的解釋是合理的。如果真的要了解，可能就要細究了。

**Comment by enoch-lightning** [comment]
不過，如果只把黑人歸類為「靈長類動物」，會讓人感覺有岐視的意味。這樣的話，程式設計必須改進，可能要把膚色納入考量，將膚色當作訓練「神經網路」的一個「特徵=feature」。

**Comment by EricAgumonJou** [comment]
我來補充一些資料給沒接觸過深度學習的人:
我們最常聽到的卷積神經網路(CNN), 是目前深度學習網路架構最常見的架構之一
該網路架構中的卷積層(Convolutional layer)及池化層(Pooling layer)可以用來強化模式辨識及相鄰資料間的關係, 
卷積神經網路的應用: 在影像、聲音等訊號類型的資料型態能得到很好的效果。

**Comment by jerry850313** [comment]
人工神經網路(Artificial Neural Network, ANN)，
一般簡稱神經網路(Neural Network, NN)或類神經網路，
基於模仿動物中樞系統的結構和功能所建構的數學模型。
圖片不太完整，可以進網址看。
![image](http://neuron.csie.ntust.edu.tw/homework/93/NN/homework2/M9304302/images/scan004.jpg)
[圖片來源](http://neuron.csie.ntust.edu.tw/homework/93/NN/homework2/M9304302/14.htm)

**Reply by xCk27x** [reply]
那請問是前授型網路還是回歸型網路比較接近人腦的連接構造?
不確定人腦的單一神經元有沒有辦法像回歸型網路一樣平行接收與輸出資訊給所有神經元

**Reply by jerry850313** [reply]
與其說是模仿單一神經元，不如說是模仿整個神經系統的運作方式。
平行接收有點像是每個感知細胞接收外界資訊。

**Reply by xCk27x** [reply]
那人腦應該比較接近回歸型網路，因為學習時感覺神經元傳遞進來的訊息應該能被各區腦神經平等的接收
再透過各區之間互相傳遞處理後的資訊統合出最後結果(類似文中所寫**收斂後的輸出值**)
